HealthcareSystem/PatientWebApp/wwwroot/js/add_feedback.js

```diff
@@ -20,9 +46,9 @@
 			return;
 		}
 
-		//I take the certain patient from the database, otherwise the currently logged in patient will be taken
+		//I take the certain patient 1309998775018 from the database, otherwise the currently logged in patient will be taken
```

Comment:
Why do this? Shouldn't you just use the jmbg of whoever is logged in?

Reply:
I corrected your suggestions except this one. If other parts are alright, my next commit will be changing AJAX calls so that they have a JWT send through a header to backend (not a JMBG through an URL).